src/libs/PolicyUtils.ts

```diff
@@ -108,6 +108,10 @@ function isExpensifyGuideTeam(email: string): boolean {
  */
 const isPolicyAdmin = (policy: OnyxEntry<Policy> | EmptyObject): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
 
+const isCollectPolicy = (policy: OnyxEntry<Policy> | EmptyObject): boolean => policy?.type === CONST.POLICY.TYPE.TEAM;
+
+const isFreePolicy = (policy: OnyxEntry<Policy> | EmptyObject): boolean => policy?.type === CONST.POLICY.TYPE.FREE;
```

Comment:
There is a policy type `Personal` also. Can a free policy be of type `free` _or_ `personal`?

https://github.com/Expensify/App/blob/f8080a2a165945814a07599e50989574c0d937e0/src/CONST.ts#L1333-L1334

Reply:
got confirmation [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1708201967994859?thread_ts=1708141365.502539&cid=C01GTK53T8Q). the current logic looks good